src/cephadm/cephadm

```diff
@@ -2452,7 +2452,9 @@ def get_container(ctx: CephadmContext,
     entrypoint: str = ''
     name: str = ''
     ceph_args: List[str] = []
-    envs: List[str] = []
+    envs: List[str] = [
+        'TCMALLOC_MAX_TOTAL_THREAD_CACHE_BYTES=134217728',
```

Comment:
hardcoded?

Reply:
> hardcoded?

It's probably fine for now. 128MB is a decent generic default seemingly both for the OSD and for RGW, but eventually I think the right long term approach is with Adam's PR here: 

https://github.com/ceph/ceph/pull/41804